### ISSUE TITLE
Add settings.xml to build

### ISSUE THREAD

**Issue by pperalta** [spring-cloud/spring-cloud-stream-binder-gemfire#4]
It should at minimum include the following:

``` xml
<repository>
    <id>gemstone-release</id>
    <name>GemStone Maven Release Repository</name>
    <url>http://dist.gemstone.com/maven/release</url>
    <snapshots>
        <enabled>true</enabled>
        <updatePolicy>always</updatePolicy>
    </snapshots>
</repository>
```

**Comment by markpollack** [comment]
Spring cloud stream and task have been retrofitted to the model in which the .settings.xml file is for IDE usage and the repositories are listed in the main pom.xml under a spring profile.  In a directory .mvn/ there can be a maven.config file that sets the default profile to be spring.  This config file is read by default for mvn 3.3 or greater or by the mvn wrapper.

**Comment by pperalta** [comment]
Fixed via de58284f9c5b8cc86ff2af4c424436431179817d
